db/migrate/20210409175743_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.uuid :uuid
```

Comment:
Please add a unique constraint for `uuid`

Reply:
Thanks, added index